guide/12-enrich-data-with-thematic-information/part1_introduction_to_geoenrichment.ipynb

```diff
@@ -11,139 +11,576 @@
     "# Part 1 - Introduction to GeoEnrichment"
```

Comment:
<p>We can load the study areas surrounding each location from a Python pickle file. The enrich capability in Business Analyst requires polygon areas to use for <a href="https://developers.arcgis.com/rest/geoenrichment/api-reference/data-apportionment.htm" target="_blank">apportioning demographic data</a> to the input geographies. The polygons delineating the area used for apportioning the data to each location are referred to as <em>study areas</em>. While, for this example, we already have study areas created, it is possible to specify parameters for study areas for the enrich tool. This is demonstrated in a later example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>